apiary-metastore-events/kafka-metastore-events/pom.xml

```diff
@@ -24,10 +24,6 @@
 
 
   <dependencies>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-    </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
```

Comment:
A decision for another potential PR - do we want to use log4j as our logging framework? I think Hive may have moved to log4j 2 while our recommended default is logback. I know there can be fun and games with dependencies and Hadoop and Hive versions so perhaps we don't want to go there now, but the Appender class you have created here would need to be rewritten if we changed logging framework so...

Reply:
Yes agreed, the logging in the whole project seems a little inconsistent. Yes the appender class will have to change slightly, we'll possibly need an additional test logging config. A further PR I think.